### PR TITLE
feat(route): Add 小米社区用户发帖

### PR DESCRIPTION
**Pull request by abc1763613206**
## Involved Issue / 该 PR 相关 Issue

Close #

## Example for the Proposed Route(s) / 路由地址示例

```routes
/miui/community/user/1200057564
/miui/community/user/95045457
```

## New RSS Route Checklist / 新 RSS 路由检查表
  
- [x] New Route / 新的路由
  - [x] Follows [Script Standard](https://docs.rsshub.app/joinus/advanced/script-standard) / 跟随 [路由规范](https://docs.rsshub.app/zh/joinus/advanced/script-standard)
- [ ] Anti-bot or rate limit / 反爬/频率限制
  - [ ] If yes, do your code reflect this sign? / 如果有, 是否有对应的措施?
- [x] [Date and time](https://docs.rsshub.app/joinus/advanced/pub-date) / [日期和时间](https://docs.rsshub.app/zh/joinus/advanced/pub-date)
  - [x] Parsed / 可以解析
  - [x] Correct time zone / 时区正确
- [ ] New package added / 添加了新的包
- [ ] `Puppeteer`

## Note / 说明

radar 规则需要抓 query 所以没做，尝试抄了一下 <https://github.com/DIYgod/RSSHub/blob/3e91889f72460ff280617f4fcd80fdb87e9ffd39/lib/routes/infzm/index.ts#L16-L17> 但测试时发现爆炸了。

### CODE REVIEW

File: lib/routes/micommunity/namespace.ts [hunk 1]
```diff
@@ -0,0 +1,6 @@
+import type { Namespace } from '@/types';
```
Comment (TonyRL) [inline lib/routes/micommunity/namespace.ts:1]: Please use namespace of
https://github.com/DIYgod/RSSHub/blob/2cbce3689e7188a9de9f8b5b082070045e5675a5/lib/routes/miui/namespace.ts#L5
instead

File: lib/routes/micommunity/user.ts [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+    },
+    features: {
+        requireConfig: false,
+        requirePuppeteer: false,
+        antiCrawler: false,
+        supportBT: false,
+        supportPodcast: false,
+        supportScihub: false,
+    },
+    radar: [],
```
Comment (TonyRL) [inline lib/routes/micommunity/user.ts:19]:
```suggestion
    radar: [
        {
            source: ['web.vip.miui.com/page/info/mio/mio/homePage'],
            target: (_, url) => `/micommunity/user/${new URL(url).searchParams.get('uid')}`,
        },
    ],
```

File: lib/routes/micommunity/namespace.ts (hunk 1 above)
Comment (abc1763613206) [reply lib/routes/micommunity/namespace.ts:1]: @TonyRL The MI community is currently in a terrible position: It's absolutely clear that it's not just MIUI that's currently included in the MI community, but also Xiaomi devices. Yet the api is still under the `miui.com` domain.
Do I need to migrate the routes under `miui/`? How do I name this route after migrating?

**Comment by abc1763613206** [comment]
ping @TonyRL 
Currently this route conflicts with the following route. 
https://github.com/DIYgod/RSSHub/blob/master/lib/routes/miui/index.ts#L6

Unfortunately, I have not found a suitable fix yet.

**Comment by TonyRL** [comment]
> ping @TonyRL Currently this route conflicts with the following route. https://github.com/DIYgod/RSSHub/blob/master/lib/routes/miui/index.ts#L6
> 
> Unfortunately, I have not found a suitable fix yet.

Change it to `/firmware/:device/:type?/:region?`

**Pull request merged**